config/lava/chromeos/tast.jinja2

```diff
@@ -0,0 +1,33 @@
+- test:
+      timeout:
+        minutes: 45
+      docker:
+        image: {{ docker_image }}
+        volumes:
+          - /tmp/octopus:/home/cros-tast/octopus
+        wait:
+          device: true
+      results:
+          location: /home/cros-tast/lava
+      definitions:
+      - repository:
+          metadata:
+            format: Lava-Test Test Definition 1.0
+            name: docker-ping
+          run:
+            steps:
+              - 'cd /home/cros-tast'
+{%- if cros_tast_tarball %}
+              - 'curl -s {{cros_tast_tarball}} | tar xjf -'
+{% endif %}
+{%- if rootfs_url %}
+              - 'curl -s {{ rootfs_url }}remote_test_runner -o remote_test_runner;chmod +x remote_test_runner'
+              - 'curl -s {{ rootfs_url }}tast -o tast;chmod +x tast'
+              - 'cp remote_test_runner /usr/bin/remote_test_runner'
+{% endif %}
```

Comment:
The `rootfs_url` parameter is used unconditionally in the `deploy` action, so it looks like it's mandatory too.

Reply:
Maybe we should leave `rootfs_url` as the actual URL for the rootfs image, and have other template parameters to download the Tast stuff.

Reply:
Just tested now and it works as designed, here is diff my test to @alexandrasp test on real hardware:
`
 - test:
       timeout:
@@ -101,9 +85,7 @@
           run:
             steps:
               - 'cd /home/cros-tast'
-              - 'curl -s http://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-amd64-generic/crostest003/remote_test_runner -o remote_test_runner;chmod +x remote_test_runner'
-              - 'curl -s http://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-amd64-generic/crostest003/tast -o tast;chmod +x tast'
-              - 'cp remote_test_runner /usr/bin/remote_test_runner'
+              - 'curl -s http://172.17.0.1/chromeos/autotest_server_package_octopus_R92_13981.0.0.tar.bz2 | tar xjf -'
 
               - 'while ! ping -c 1 -w 1 $(lava-target-ip); do sleep 1; done'
               - 'ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /home/cros-tast/.ssh/id_rsa root@$(lava-target-ip) whoami'


`